### DISCUSSION TITLE
Specific pipeline selection

### DISCUSSION THREAD

**Discussion by pvlcor** [profclems/glab#704] [Q&A] [answered]
Is there a way to select a pipeline by id? Smth like
`glab ci view 23453243`

**Comment by zemzale** [comment, marked as answer]
ATM it's not possible, but it has been brought up and is gonna be added soon probably 
See: https://github.com/profclems/glab/issues/699

**Reply by pvlcor** [reply]
Super, thanks!

**Comment by pvlcor** [comment]
https://github.com/profclems/glab/issues/699#issuecomment-833663309
